frontend_csv/src/scanner/scanner.rs

```diff
@@ -1,159 +1,189 @@
+                CSVToken {
+                    ty: CSVTokenType::ShortChannelId,
+                    val: "short_channel_id".to_string(),
+                },
+            ),
+            (
+                "sha256".to_string(),
+                CSVToken {
+                    ty: CSVTokenType::Sha256,
+                    val: "sha256".to_string(),
+                },
+            ),
+            (
+                "u16".to_string(),
+                CSVToken {
+                    ty: CSVTokenType::U16,
+                    val: "u16".to_string(),
+                },
+            ),
+            (
+                "u32".to_string(),
+                CSVToken {
+                    ty: CSVTokenType::U32,
+                    val: "u32".to_string(),
+                },
+            ),
+            (
+                "u64".to_string(),
+                CSVToken {
+                    ty: CSVTokenType::U64,
+                    val: "u64".to_string(),
+                },
+            ),
+            (
+                "channel_id".to_string(),
+                CSVToken {
+                    ty: CSVTokenType::ChannelId,
+                    val: "channel_id".to_string(),
+                },
+            ),
+            (
+                "signature".to_string(),
+                CSVToken {
+                    ty: CSVTokenType::Signature,
+                    val: "signature".to_string(),
+                },
+            ),
+            (
+                "point".to_string(),
+                CSVToken {
+                    ty: CSVTokenType::Point,
+                    val: "point".to_string(),
+                },
+            ),
+            (
+                "chain_hash".to_string(),
+                CSVToken {
+                    ty: CSVTokenType::ChainHash,
+                    val: "chain_hash".to_string(),
+                },
+            ),
+            (
+                "byte".to_string(),
+                CSVToken {
+                    ty: CSVTokenType::Byte,
+                    val: "byte".to_string(),
+                },
+            ),
+            (
+                "bigsize".to_string(),
+                CSVToken {
+                    ty: CSVTokenType::BigSize,
+                    val: "bigsize".to_string(),
+                },
+            ),
+            (
+                ",".to_string(),
+                CSVToken {
+                    ty: CSVTokenType::Comma,
+                    val: ",".to_string(),
+                },
+            ),
+        ]);
 
-            match _symbol.to_string().as_str() {
-                "," | "\n" => {
-                    if !current_buffer.is_empty() {
-                        tokenize.push_back(CSVToken {
-                            _type: CSVTokenType::LiteralString,
-                            _value: current_buffer.clone(),
+        return Scanner { line, keywords };
+    }
+
+    pub fn scan(&mut self, _symbols: &Vec<char>) -> Vec<CSVToken> {
```

Comment:
```suggestion
    pub fn scan(&mut self, symbols: &Vec<char>) -> Vec<CSVToken> {
```

can you remove all the `_`?